### PR TITLE
49 add suitable api

### PR DESCRIPTION
**Pull request by AlexKay28**
- add new api format
- change configure endpoint name to login
- update tests
- response status handling

### CODE REVIEW

File: backend/api/answer_api.py [hunk 1]
```diff
@@ -6,52 +6,64 @@
...
-            "question": first_language_phrase,
-            "answer": second_language_phrase,
-            "answer_user": second_language_phrase_answer,
-            "is_equal": is_equal,
-            "score": equality_rate,
-            "differences": differences,
-        }
-    )
+    try:
+        if request.method == "PATCH":
```
Comment (alantishin) [inline backend/api/answer_api.py:17]: Does row with `@api.route("/answer", methods=["PATCH"])` not enough for checking request type?

Comment (AlexKay28) [reply backend/api/answer_api.py]: you are right. This was unneeded. I have removed this from now.

File: tests/conftest.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from uuid import UUID
 from flask import Flask, jsonify, request
 
-sys.path.append("../langflow")
+sys.path.append("/home/alexkay/mygit/langflow/backend")
```
Comment (alantishin) [inline tests/conftest.py:6]: aaaaaaaaaaaaaaaaaa!!!!
WTF is this?

Comment (AlexKay28) [reply tests/conftest.py]: sorry. fixed

**Review by alantishin — APPROVED** [review]

**Pull request merged**